data/maps/SP_Scenarios/Along The River.wmf/scripting/objectives.lua

```diff
@@ -0,0 +1,84 @@
+-- =======================================================================
+--                             Along The River
+-- =======================================================================
+
+obj_defeat_red = {
+   name = "defeat_red",
+   title = _"Defeat Törver",
+   number = 1,
+   body = objective_text(_"Defeat the armies of Törver.",
+      li_image("map:genstats_enable_plr_02.png", _[[]])
+   ),
+}
+
+obj_build_port = {
+   name = "build_port",
+   title = _"Build a port",
+   number = 1,
+   body = objective_text(_"Build a port in the land of Törver.",
+      li_image("tribes/buildings/warehouses/barbarians/port/menu.png", _[[]])
+   ),
+}
+
+obj_conquer_pirate_sentry = {
+   name = "conquer_pirate_sentry",
+   title = _"Conquer pirate sentry",
+   number = 1,
+   body = objective_text(_"Conquer the pirate sentry near the brook to the east.",
+      li_image("tribes/buildings/militarysites/barbarians/sentry/menu.png", _[[]])
+   ),
+}
+
+obj_build_warehouse = {
+   name = "build_warehouse",
+   title = _"Build warehouse",
+   number = 1,
+   body = objective_text(_"Build a warehouse close to the pirate sentry near the brook.",
+      li_image("tribes/buildings/warehouses/barbarians/warehouse/menu.png", _[[]])
+   ),
+}
+
+obj_bring_shovels_1 = {
+   name = "bring_shovels_1",
+   title = _"Bring shovels",
+   number = 1,
+   body = objective_text(_"Bring 4 shovels to the warehouse near the brook.",
```

Comment:
Should "4" be spelled out as "four"? I think we had a rule to spell out numbers less than 13 but I can never remember that one exactly ;)